src/apps/answers/service.py

```diff
@@ -366,7 +367,19 @@ async def validate_multiinformant_assessment(
             activity, flow = await asyncio.gather(activity_future, flow_future)
             if not activity and not flow:
                 raise MultiinformantAssessmentInvalidActivityOrFlow()
-
+            
+        try:
+            await CheckAccessService(self.session, self.user_id).check_applet_manager_list_access(applet_id)
+            is_admin = True
+        except AppletAccessDenied:
+            is_admin = False
+
+        if not is_admin:
+            if not target_subject or not source_subject:
+                raise MultiinformantAssessmentNoAccessApplet("Missing target subject or source subject")
+            
+        await self._validate_relation_between_subjects_in_applet(respondent_subject, target_subject, source_subject)
```

Comment:
We want to explicitly prevent team members who are not owners or managers from initiating take now. Since all team members also have a role of respondent in each applet, currently coordinators, editors, and reviewers would be able to pass this check.

We should also add a check that the logged in user is not otherwise a team member by fetching all the roles held by them, and making sure they don't include any team roles besides the two allowed ones.

Reply:
I agree with @sultanofcardio